okta/data_source_okta_app_oauth.go

```diff
@@ -195,6 +202,35 @@ func dataSourceAppOauthRead(ctx context.Context, d *schema.ResourceData, m inter
 	if err != nil {
 		return diag.Errorf("failed to list OAuth's app groups and users: %v", err)
 	}
+	skipClientSecrets := false // Do we ever need to skip doing this?
```

Comment:
there's an `omit_secret` attribute in the resource, we could follow the same pattern here?

https://github.com/okta/terraform-provider-okta/blob/090c4743242013bed8a5f612c639eacc798f5374/okta/resource_okta_app_oauth.go#L146-L152

https://github.com/okta/terraform-provider-okta/blob/090c4743242013bed8a5f612c639eacc798f5374/okta/resource_okta_app_oauth.go#L532-L534